.github/workflows/push-auto-generate.yml

```diff
@@ -53,6 +55,27 @@ jobs:
           cd $SAMPLEFOLDER_PATH
           brm generate
 
+      - name: Bump version and push tag
+        id: tag_version
+        uses: mathieudutour/github-tag-action@v6.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          tag_prefix: "${{ needs.get-module-to-validate.outputs.module_dir }}/"
+          dry_run: true
+          default_bump: false
+
+      - name: Set Configurations
+        if: steps.tag_version.outputs.release_type
+        shell: bash
+        env:
+          SAMPLEFOLDER_PATH: ${{ needs.get-module-to-validate.outputs.module_dir }}
+          OLD_VERSION: ${{ steps.tag_version.outputs.previous_version }}
+          VERSION: ${{ steps.tag_version.outputs.new_version }}
+        run: |
+          cd $SAMPLEFOLDER_PATH
+
+          sed -ri "s/$OLD_VERSION/$VERSION/" "README.md" || exit 1
+
```

Comment:
Sorry, I renamed the file, so you might need to merge main and resolve the conflict.

Reply:
rolling the changes to this file back for the moment, they need more testing.